bpython/curtsiesfrontend/repl.py

```diff
@@ -55,6 +60,8 @@
 )
 from ..translations import _
 
+InputOrOutput = Union[Literal["input"], Literal["output"]]
```

Comment:
I would prefer to avoid the runtime dependency on `typing_extensions`. `InputOrOutput` could be an enum instead.

Reply:
there's already an enum and it even says
> (use LineTypeTranslator.INPUT or LineTypeTranslator.OUTPUT to avoid typing these strings)
a line above one of these that it changed! I just didn't use it.